### DISCUSSION TITLE
Is there a way to import current images

### DISCUSSION THREAD

**Discussion by Taylorjoey03** [SergiX44/XBackBone#575] [General]
Hi.

I am just installing xBackBone after using ShareX for a while but I wanted a proper interface to manage it. The issue is I can not get all my old images (3033 images) to import automatically by having them in my storage folder or the users storage folder.

I know there is a feature for "orphaned files", I was wondering how I could possibly use this or any other way to add the files without having to manually upload all the files via ShareX.

**Comment by SenkaWolf** [comment]
The only way is to upload them via the web interface.

If you want to be extra fancy but probably overkill is to upload the images manually then write a SQL script to adjust the dates etc so they don't all just show as the day you migrated to XBackBone. I will be doing a write up of what I did but for the average person it's not worth the effort.
